run_inference.py

```diff
@@ -71,7 +71,7 @@ def main():
     for ext in args.img_exts:
         test_files = data_dir.files('*1.{}'.format(ext))
         for file in test_files:
-            img_pair = file.parent / (file.namebase[:-1] + '2.{}'.format(ext))
+            img_pair = file.parent / (file.basename().splitext()[0][:-1] + '2.{}'.format(ext))
```

Comment:
I think the right term here is "stem". see https://pathpy.readthedocs.io/en/latest/api.html#path.Path.stem (it was called namebase before)